viite-UI/src/view/RoadLayer.js

```diff
@@ -67,38 +67,38 @@
               'Ajorata:&nbsp;' + roadData.trackCode + '<br>' +
               'AET:&nbsp;' + roadData.startAddressM + '<br>' +
               'LET:&nbsp;' + roadData.endAddressM + '<br>' +
-              'Tietyyppi:&nbsp;' + displayRoadType(roadData.roadTypeId) + '<br>';
+              'Hallinnollinen luokka:&nbsp;' + displayAdministrativeClass(roadData.administrativeClassId) + '<br>';
           }
         }
       }
       overlay.setPosition(coordinate);
     };
 
-    var displayRoadType = function (roadTypeCode) {
-      var roadType;
-      switch (roadTypeCode) {
-        case LinkValues.RoadTypeShort.PublicRoad.value:
-          roadType = LinkValues.RoadTypeShort.PublicRoad.description;
+    var displayAdministrativeClass = function (administrativeClass) {
+      var administrativeClass;
+      switch (administrativeClass) {
+        case LinkValues.AdministrativeClassShort.PublicRoadClass.value:
+          administrativeClass = LinkValues.AdministrativeClassShort.PublicRoadClass.description;
           break;
-        case LinkValues.RoadTypeShort.FerryRoad.value:
-          roadType = LinkValues.RoadTypeShort.FerryRoad.description;
+        case LinkValues.AdministrativeClassShort.FerryRoad.value:
```

Comment:
This does not exist

Reply:
Yep, so do not the other two, a few lines downward 👍